build/WinObjC.Frameworks/WinObjC.Frameworks.nuproj

```diff
@@ -90,7 +96,6 @@
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\MediaPlayer\dll\MediaPlayer.vcxproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\MessageUI\dll\MessageUI.vcxproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\Metal\dll\Metal.vcxproj" />
-    <ProjectReference Include="$(MSBuildThisFileDirectory)..\MobileCoreServices\dll\MobileCoreServices.vcxproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\OpenGLES\dll\OpenGLES.vcxproj" />
```

Comment:
Should this be removed?  The mobilecoreservices project doesn't have much in it, but, I think we should keep building it.